modules/ai/cognitiveservices/main.bicep

```diff
@@ -137,9 +137,12 @@ resource cognitiveService 'Microsoft.CognitiveServices/accounts@2023-05-01' = {
 
 @batchSize(1)
 resource cognitiveServiceDeployment 'Microsoft.CognitiveServices/accounts/deployments@2023-05-01' = [for (deployment, index) in deployments: {
```

Comment:
Can we change deployments from a generic array to an array of a a user-defined type?

Reply:
@dciborow - I created a new issue to track that change https://github.com/Azure/bicep-registry-modules/issues/516